pypesto/objective/amici_calculator.py

```diff
@@ -23,6 +25,7 @@ class AmiciCalculator:
     Class to perform the actual call to AMICI and obtain requested objective
     function values.
     """
+    _known_least_squares_safe = False
```

Comment:
this way, it becomes a class attribute, not an object attribute ... might be better to define inside init